Jelly/Classes/private/Animators/Animated Animators/CoverAnimator.swift

```diff
@@ -56,6 +57,19 @@ extension CoverAnimator : UIViewControllerAnimatedTransitioning {
         let propertyAnimator = UIViewPropertyAnimator(duration: animationDuration, timingParameters: timingCurveProvider)
 
         propertyAnimator.addAnimations {
+            if isPresentation {
+                if let presentingViewController = transitionContext.viewController(forKey: .from) {
+                    presentingViewController.view.transform = CGAffineTransform(scaleX: self.presentation.depthScale,
+                                                                                y: self.presentation.depthScale)
+                    if UIApplication.shared.delegate?.window??.safeAreaInsets.top ?? 0 > 20 {
```

Comment:
Could you explain why are you checking the Safe-Area Insets over here?

Reply:
This is checking for any X device to round the corners for the effect